libraries/AP_HAL_ChibiOS/hwdef/QioTekZealotF427/hwdef-bl.dat

```diff
@@ -0,0 +1,49 @@
+# hw definition file for processing by chibios_hwdef.py
+# for for QioTekZealotF427 hardware bootloader from Qio-tek.com 
+
+MCU STM32F4xx STM32F427xx
+
+APJ_BOARD_ID 1021
+
+OSCILLATOR_HZ 24000000
+
+STM32_ST_USE_TIMER 5
+
+# flash size
+FLASH_SIZE_KB 2048
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+FLASH_BOOTLOADER_LOAD_KB 128
```

Comment:
why do you reserve so much space for the bootloader? The minimum would be 16k. Using 128k wastes quite a lot of flash space. You have plenty now, but it does mean as we get close to filling 2MByte you won't be able to have as many features

Reply:
Flash after the current firmware compiled a lot free. so I kept the boot loader 128K. but I will I will modify it.